### PR TITLE
CLDR-15034 kbd: doc naming updates from Sep 12/13, 2022

### PR DESCRIPTION
**Pull request by srl295**
- updated docs with naming changes from #2354
- also improved the transform docs, but more to do there

CLDR-15034

### CODE REVIEW

File: docs/ldml/tr35-keyboards.md [hunk 1]
```diff
@@ -168,7 +167,7 @@ Note that in parts of this document, the format `@x` is used to indicate the _at
 Keyboard use can be challenging for individuals with various types of disabilities. For this revision, the committee is not evaluating features or architectural designs for the purpose of improving accessibility. Such consideration could be fruitful for future revisions. However, some points on this topic should be made:
 
 1. Having an industry-wide standard format for keyboards will enable accessibility software to make use of keyboard data with a reduced dependence on platform-specific knowledge.
-2. Some features, such as multitap and flicks, have the potential to reduce accessibility and thus should be discouraged. For example, multitap requires pressing keys at a certain speed, and flicks require a more complex movement (press-and-flick) beyond a simple tap.
+2. Some features, such as multiTap and flicks, have the potential to reduce accessibility and thus should be discouraged. For example, multiTap requires pressing keys at a certain speed, and flicks require a more complex movement (press-and-flick) beyond a simple tap.
```
Comment (mcdurdin) [inline docs/ldml/tr35-keyboards.md:170]:
```suggestion
2. Some features, such as multiTap and flicks, have the potential to reduce accessibility and thus should be discouraged. For example, multiTap requires pressing keys at a certain speed, and flicks require a more complex movement (press-and-flick) beyond a simple tap. Alternatively, inclusion of accessible methods of generating the same outputs (for example, simple keys on an additional layer), should be considered.
```

File: docs/ldml/tr35-keyboards.md [hunk 2]
```diff
@@ -972,14 +971,14 @@ The following elements are not imported from the source file:
...
+The displays can be used to describe what is to be displayed on the keytops for various keys. For the most part, such explicit information is unnecessary since the `@to` element from the `keys/key` element can be used. But there are some characters, such as diacritics, that do not display well on their own and so explicit overrides for such characters can help.
 Another useful scenario is where there are doubled diacritics, or multiple characters with spacing issues.
 
-The `displayMap` consists of a list of display subelements.
+The `displays` consists of a list of display subelements.
 
-displayMaps are designed to be shared across many different keyboard layout descriptions, and imported with `<import>` in where needed.
+displayss are designed to be shared across many different keyboard layout descriptions, and imported with `<import>` in where needed.
```
Comment (mcdurdin) [inline docs/ldml/tr35-keyboards.md:981]:
```suggestion
`displays` elements are designed to be shared across many different keyboard layout descriptions, and imported with `<import>` in where needed.
```

The double-s may have been intentional but very confusing!

File: docs/ldml/tr35-keyboards.md [hunk 3]
```diff
@@ -1056,34 +1055,34 @@ _Attribute:_ `display` (required)
...
         <display to="\u0300" display="ˋ" /> <!-- \u02CB -->
         <display to="\u0301" display="ˊ" /> <!-- \u02CA -->
         <display id="shift"  display="⇪" /> <!-- U+21EA -->
-    </displayMap>
+    </displays>
 </keyboard>
 ```
 
-To allow `displayMap`s to be shared across keyboards, there is no requirement that `@to` in a `display` element matches any `@to`/`@id` in any `keys/key` element in the keyboard description.
+To allow `displays`s to be shared across keyboards, there is no requirement that `@to` in a `display` element matches any `@to`/`@id` in any `keys/key` element in the keyboard description.
```
Comment (mcdurdin) [inline docs/ldml/tr35-keyboards.md:1066]:
```suggestion
To allow `displays` elements to be shared across keyboards, there is no requirement that `@to` in a `display` element matches any `@to`/`@id` in any `keys/key` element in the keyboard description.
```

File: docs/ldml/tr35-keyboards.md [hunk 4]
```diff
@@ -1268,12 +1276,12 @@ This example shows some of the mappings for a French keyboard layout:
...
-		<vkeyMap from="Q" to="A" />
-		<vkeyMap from="W" to="Z" />
-		<vkeyMap from="A" to="Q" />
-		<vkeyMap from="Z" to="W" />
-    </vkeyMaps>
+    <vkeys>
+		<vkey from="Q" to="A" />
+		<vkey from="W" to="Z" />
+		<vkey from="A" to="Q" />
+		<vkey from="Z" to="W" />
```
Comment (mcdurdin) [inline docs/ldml/tr35-keyboards.md:1283]:
```suggestion
		  <vkey from="Q" to="A" />
		  <vkey from="W" to="Z" />
		  <vkey from="A" to="Q" />
		  <vkey from="Z" to="W" />
```

File: docs/ldml/tr35-keyboards.md [hunk 5]
```diff
@@ -1490,14 +1499,14 @@ The reordering algorithm consists of four parts:
    * The **quaternary weight** is the index of the character in the string. This is solely to ensure a stable sort for sequences of characters with the same tertiary weight.
 2. Mark each character as to whether it is a prebase character, one that is typed before the base and logically stored after. Thus it will have a primary order > 0.
 3. Use the sort key and the prebase mark to identify runs. A run starts with a prefix that contains any prebase characters and a single base character whose primary and tertiary key is 0. The run extends until, but not including, the start of the prefix of the next run or end of the string.
-   * `run := prebase* (primary=0 && tertiary=0) ((primary≠0 || tertiary≠0) && !prebase)*`
+   * `run := preBase* (primary=0 && tertiary=0) ((primary≠0 || tertiary≠0) && !prebase)*`
```
Comment (mcdurdin) [inline docs/ldml/tr35-keyboards.md:1502]:
```suggestion
   * `run := preBase* (primary=0 && tertiary=0) ((primary≠0 || tertiary≠0) && !preBase)*`
```

**Review by mcdurdin — CHANGES_REQUESTED** [review]
Looking good. Minor edits

File: docs/ldml/tr35-keyboards.md (hunk 1 above)
Comment (srl295) [reply docs/ldml/tr35-keyboards.md:170]: +1000

File: docs/ldml/tr35-keyboards.md (hunk 2 above)
Comment (srl295) [reply docs/ldml/tr35-keyboards.md:981]: no, it was search and replace… (displayMaps)s -> (displays)s

File: docs/ldml/tr35-keyboards.md [hunk 6]
```diff
@@ -978,7 +978,7 @@ Another useful scenario is where there are doubled diacritics, or multiple chara
 
 The `displays` consists of a list of display subelements.
 
-displayss are designed to be shared across many different keyboard layout descriptions, and imported with `<import>` in where needed.
+`displays` elements are designed to be shared across many different keyboard layout descriptions, and imported with `<import>` in where needed.
```
Comment (mcdurdin) [inline docs/ldml/tr35-keyboards.md:981]:
```suggestion
`displays` elements are designed to be shared across many different keyboard layout descriptions, and imported with `<import>` where needed.
```

Missed that last time...

**Review by mcdurdin — APPROVED** [review]
One more grammar typo!

**Pull request merged**